http/notification_rule.go

```diff
@@ -146,9 +147,13 @@ type notificationRuleLinks struct {
 
 type notificationRuleResponse struct {
 	influxdb.NotificationRule
-	Labels []influxdb.Label      `json:"labels"`
-	Links  notificationRuleLinks `json:"links"`
-	Status string                `json:"status"`
+	Labels          []influxdb.Label      `json:"labels"`
+	Links           notificationRuleLinks `json:"links"`
+	Status          string                `json:"status"`
+	LatestCompleted string                `json:"latestCompleted,omitempty"`
```

Comment:
You can just use time.Time for this field. It will work fine

Reply:
Hey @kelwang - I'm really new to go so my apologies if this is a straightforward SO answer, but I couldn't seem to format the dates using `https://github.com/influxdata/influxdb/pull/16325/files/8a7cd82dc1ad34dcc775b58e6af4526079dc7704#diff-0045aab5f4f34cfeb0f1646f5710df0aR213-R214` when the data was typecast as `time.Time`. Is there something i'm missing?

Reply:
yeah, that's the default format when you do json encoding of time.Time. see https://golang.org/src/time/time.go#L1264. You can see the example here: https://play.golang.org/p/6sozCf40hMh

Reply:
Aw perfect, thanks for the heads up!